stylesheets/postcss.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = (ctx) => ({
+  plugins: {
+    cssnano: {
+      preset: [
+        'default',
+        {
+          discardComments: { exclude: true },
+          minifySelectors: { exclude: true }, // this one we'll do ourselves
+          minifyFontValues: { exclude: true },
+          mergeRules: { exclude: true }, // TODO reenable
+          uniqueSelectors: { exclude: true },
+          cssDeclarationSorter: { exclude: true },
```

Comment:
I think we should explain why each rule are disabled. Do we want to re-enable `discardComments`, `minifyFontValues`, `uniqueSelectors` and `cssDeclarationSorter`? If not, why?

[`cssDeclarationSorter`](https://cssnano.co/docs/optimisations/cssdeclarationsorter/) is probably not worth it, since, as far as I understand, the benefit would be relatively low. 

We could also add a link on the documentation:

- https://cssnano.co/docs/optimisations/minifyfontvalues/ 
- https://cssnano.co/docs/optimisations/cssdeclarationsorter/
- ...

Reply:
The main reason for tuning the rules is to be able to start with a 1-to-1 match with what we have now. We don't want this step (the switch to PostCSS+cssnano) to introduce any changes. We want to take those changes in as separate commits.

* discardComments: We only have three comments at the top and we want to keep those; if we add comments elsewhere, this will need to be adjusted
* minifyFontValues switches to numeric weights and I don't like that because it makes the stylesheet harder to build on
* cssDeclarationSorter is pointless for minification; it's just sorting the property names, which introduces HUGE changes
* uniqueSelectors: this is just adding a lot of changes, so we're not ready for it yet...but something to consider

> We could also add a link to the documentation

:+1: